config/samples/_v1_checkpointrestoreoperator.yaml

```diff
@@ -15,15 +15,25 @@ spec:
     maxCheckpointsPerNamespace: 50
     maxCheckpointsPerPod: 30
     maxCheckpointsPerContainer: 10
+    maxCheckpointSize: 10
+    maxTotalSizePerNamespace: 1000
+    maxTotalSizePerPod: 500
+    maxTotalSizePerContainer: 100
   # containerPolicies:
   #   - namespace: <namespace>
   #     pod: <pod_name>
   #     container: <container_name>
   #     maxCheckpoints: 5
+  #     maxCheckpointSize: 10
+  #     maxTotalSize: 100
```

Comment:
It might be better to use format similar to other resource limits in Kubernetes:

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory


> Limits and requests for memory are measured in bytes. You can express memory as a plain integer or as a fixed-point number using one of these [quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/) suffixes: E, P, T, G, M, k. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.